keycloak/keycloak-ui
# User events configuration fine tuning - #1562

## Motivation
Closes #1562

## Verification Steps

1. Go to `Realm settings >> Events >> User events settings`.
2. Verify that the header title `User events configuration` has been removed.
3. Set `Save events` to `On` and observe that the list of events only displays when the button `Save` is clicked.
4. In the events list, select the `three dots` button in the first row and verify that button is now reading `Remove`.
5. Go to `Admin events settings` and verify that the header title `Admin events configuration` has been removed.

## Checklist:
